textproc/epubcheck/Portfile

```diff
@@ -4,35 +4,38 @@ PortSystem              1.0
 PortGroup               java 1.0
 
 name                    epubcheck
-version                 1.2
+version                 4.2.2
 categories              textproc
 platforms               darwin
-license                 BSD 
+license                 BSD 3-Clause \"New\" or \"Revised\" License
 maintainers             nomaintainer
 supported_archs         noarch
 
-description             ePub Validator to validate ePub documents
+description             EPUB validator
 
-long_description        EpubCheck is a tool to validate IDPF  EPUB files. \
-                        It can detect many types of errors in EPUB. OCF \
-                        container structure, OPF and OPS mark-up, and \
-                        internal reference consistency are checked.
+long_description        EPUBCheck is a tool to validate the \
+                        conformance of EPUB publications against the \
+                        EPUB specifications. EPUBCheck can be run as a \
+                        standalone command-line tool or used as a Java \
+                        library.
 
-homepage                https://code.google.com/p/epubcheck/
-master_sites            googlecode
+homepage                https://github.com/w3c/epubcheck
+master_sites            https://github.com/w3c/epubcheck/releases/download/v${version}
 
 distname                ${name}-${version}
 
 use_zip                 yes
 
-checksums               sha256  2ad436aeaaa341f28e11fdb91256bc04e8b0747d50857f58f5bd97ec6a6cb265 \
-                        rmd160  a086609ba13ae36ec63759cf2f7f474b88c8ccdc \
-                        size    1399350
+checksums               sha256  029ee9cda237a42e0d4c1a2b8771a440a4a6326449db843510b16a3c61045463 \
+                        rmd160  cebf60866877013c2a9170605548d139fd16a232 \
+                        size    11990465
 
-java.version            1.5+
-java.fallback           openjdk11
+java.version            1.7+
+java.fallback           openjdk13
 
-extract.mkdir           yes
+depends_run             bin:java:kaffe
```

Comment:
Please do not add a fallback dependency on `kaffe`; that port is deprecated and currently broken, and any ports using it need to be moved away from it. I specifically removed this dependency from `epubcheck` in #5174 because it is unnecessary when using the Java portgroup to specify an OpenJDK fallback.